seccomp_test.go

```diff
@@ -519,3 +531,64 @@ func TestRuleAddAndLoad(t *testing.T) {
 		t.Errorf("Syscall returned incorrect error code - likely not blocked by Seccomp!")
 	}
 }
+
+func TestLogAct(t *testing.T) {
```

Comment:
Can we skip this if the libseccomp version we build against is too low to have support? We have a function to grab library version we built against, it should be possible to skip the test if it's detected as too low.

Reply:
Yes! Good idea. Unfortunately, it doesn't solve the case where libseccomp is "new" but the kernel is "old" but at least it improves the situation.

Reply:
The new API level support in libseccomp allows us to handle the case that I mentioned above. The updated PR now handles all combinations of new/old libseccomp-golang, new/old libseccomp, and new/old kernel.